src/main/kotlin/io/libp2p/security/noise/NoiseXXSecureChannel.kt

```diff
@@ -202,7 +160,50 @@ class NoiseXXSecureChannel(val localKey: PrivKey, val localDHState: DHState, val
                 ctx.fireUserEventTriggered(secureChannelInitialized)
                 return
             }
-            super.channelRead(ctx, msg1)
         }
+
+        override fun channelRegistered(ctx: ChannelHandlerContext?) {
```

Comment:
You can safely remove `?` from param type

Reply:
done